arbor/morph/region.cpp

```diff
@@ -551,6 +552,73 @@ std::ostream& operator<<(std::ostream& o, const named_& x) {
     return o << "(region \"" << x.name << "\")";
 }
 
+// Region with all segments with projection less than val
```

Comment:
This comment was probably copied from another region

Reply:
Yes. There is also a cut and paste comment  2 lines lower.
I will fix it.